### PR TITLE
updated to work with winter smith ~2.0.5 and grunt ~0.4.0

### PR DESCRIPTION
**Pull request by samharnack**

### CODE REVIEW

**Comment by ajpiano** [comment]
Whoa, other people are actually using this? Sweet.

File: tasks/wintersmith.js [hunk 1]
```diff
@@ -1,33 +1,29 @@
...
     });
-  });
 
-  grunt.registerHelper("wintersmith", function(options, callback) {
-      wintersmith( options, callback );
   });
 
-  grunt.registerHelper("wintersmith-tree", function(options, callback) {
-    wintersmith.loadContents( options.contents, callback );
+  grunt.registerHelper("wintersmith", function (options, callback) {
```
Comment (ajpiano) [inline tasks/wintersmith.js:21]: Isn't `registerHelper` deprecated in 0.4?

Comment (samharnack) [inline tasks/wintersmith.js:21]: Apparently it is... I'll update it.

**Comment by samharnack** [comment]
Haha. Yea! Thanks for doing all the hard work... I'm new to the wintersmith framework and have a compulsion for a clean tool kit. :)

**Comment by davidtucker** [comment]
Can we get this pull request integrated now?

**Comment by ajpiano** [comment]
@davidtucker, have at it - I just transferred the repository to you!

**Comment by davidtucker** [comment]
Thanks!

**Pull request merged**